master/internal/api_master.go

```diff
@@ -241,3 +241,28 @@ func (a *apiServer) ResourceAllocationAggregated(
 
 	return a.m.fetchAggregatedResourceAllocation(req)
 }
+
+func (a *apiServer) CleanupLogs(
+	ctx context.Context, req *apiv1.CleanupLogsRequest,
+) (*apiv1.CleanupLogsResponse, error) {
+	u, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	permErr, err := cluster.AuthZProvider.Get().CanUpdateMasterConfig(ctx, u)
```

Comment:
is update master config the right attribute for this?

Reply:
good question, should we create a new attribute for this?

Reply:
Any thoughts on this @NicholasBlaskey ? My thinking was is that if someone can update the master config they will be able to modify the log retention schedule, so should be able to trigger a manual run. Is there a better permission this could fall under?

Reply:
I think ```CanUpdateMasterConfig``` is fine with a comment now. 

I think we should end up replacing all RBAC interfaces that boil down to ```IsClusterAdmin``` to just that. I think that should be a separate ticket.

This really only matters when custom roles happen which I don't see any desire to do that soon.